pkg/runtime/pubsub/bulksub.go

```diff
@@ -84,11 +81,12 @@ func (p *defaultBulkSubscriber) BulkSubscribe(ctx context.Context, req contribPu
 			bulkMsg.ContentType = *msg.ContentType
 		}
 
+		done := make(chan struct{})
 		msgCbChan <- msgWithCallback{
 			msg: bulkMsg,
 			cb: func(ierr error) {
 				err = ierr
-				done <- struct{}{}
+				close(done)
```

Comment:
The channel was not being closed before, causing a leak. In this case, I am closing the channel here, which will also cause the `<- done` to continue.

Reply:
Just picking brains here, is it really required to close a channel though? https://groups.google.com/g/golang-nuts/c/pZwdYRGxCIk/m/qpbHxRRPJdUJ